### ISSUE TITLE
float 0.9.0 segfaults in the GAP master branch

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/float#34]
float 0.9.0 first displays the following error message:
```
InstallOtherMethod( VALUE_GLOBAL( i[1] ), "MPFR float", [ IsMPFRFloat ], i[2]
 ); at /home/gap/inst/gap-master/pkg/float-0.9.0/lib/mpfr.gi:101
```
and then segfaults. I have disabled 0.9.0 for tests in the GAP master branch, replacing it by float-0.8.0: https://github.com/gap-system/gap-distribution/commit/e00baed49e3c12453800bd753aee9bbcaf2d8bfd

**Comment by alex-konovalov** [comment]
P.S. see e.g. this Travis CI build: https://travis-ci.org/gap-system/gap-docker-pkg-tests-master/jobs/317827235

**Comment by alex-konovalov** [comment]
This no longer happens, since it happened that float-0.9.0 was requiring https://github.com/gap-system/gap/pull/2016 to be merged.

What happened:
- float 0.9.0 has been released and picked up before matching PR https://github.com/gap-system/gap/pull/2016 has been merged.
- I've noticed that it did not work and disabled float-0.9.0, requiring 0.8.0 for tests
- then merged the PR, and discovered that now 0.8.0 no longer works
- then enabled 0.9.0 back again, and tests went back ti normal.

In the future, it will be nice to coordinate better changes that require simultaneous changes in teh core system and packages :)
